docs/drafts/app-orchestrator/app-builder.md

```diff
@@ -0,0 +1,34 @@
+# AppBuilder
+
+`AppBuilder` allows to compose different pieces into a single Oryx application.
```

Comment:
can we introduce the changing concept here?

Reply:
What do you mean by "changing concept"?

Reply:
I meant _chaining_